node/tessel-export.js

```diff
@@ -755,10 +755,9 @@ Tessel.Pin.prototype.analogWrite = function(val) {
     throw new RangeError('Analog write can only be used on Pin 7 (G3) of Port B.');
   }
 
-  // v_dac = data/(0x3ff)*reference voltage
-  var data = val / 3.3 * 0x3ff;
-  if (data > 1023 || data < 0) {
-    throw new RangeError('Analog write must be between 0 and 3.3');
+  var data = val * 0x3ff;
+  if (data > 0x3ff || data < 0) {
```

Comment:
This is a minor, but important and valuable readability change—I'm glad you spotted it.